modular_zubbers/code/datums/quirks/positive_quirks/venemous_bite/venomous_bite_quirk.dm

```diff
@@ -28,21 +28,43 @@
 	category = PREFERENCE_CATEGORY_MANUALLY_RENDERED
 	savefile_key = "venomous_bite_venom"
 	savefile_identifier = PREFERENCE_CHARACTER
-	/// Format: (reagent typepath -> list(amount to inject per bite, cooldown))
+	/// Format: (reagent typepath -> list(amount to inject per bite, cooldown, can be milked (venom_milker.dm)))
 	var/static/list/venomous_bite_choice_specs = list(
-		/datum/reagent/toxin = list(5, 80 SECONDS),
-		/datum/reagent/toxin/venom = list(5, 180 SECONDS),
-		/datum/reagent/toxin/carpotoxin = list(5, 60 SECONDS), // less powerful than toxin
+		/datum/reagent/toxin = list(5, 80 SECONDS, FALSE),
+		/datum/reagent/toxin/venom = list(5, 180 SECONDS, FALSE),
+		/datum/reagent/toxin/carpotoxin = list(5, 60 SECONDS, FALSE), // less powerful than toxin
 		// drugs
-		/datum/reagent/drug/space_drugs = list(5, 60 SECONDS),
-		/datum/reagent/toxin/mindbreaker = list(5, 60 SECONDS),
+		/datum/reagent/drug/space_drugs = list(5, 60 SECONDS, TRUE),
+		/datum/reagent/toxin/mindbreaker = list(5, 60 SECONDS, FALSE),
 		// erp stuff
-		/datum/reagent/drug/aphrodisiac/crocin = list(5, 2 SECONDS),
-		/datum/reagent/drug/aphrodisiac/crocin/hexacrocin = list(5, 5 SECONDS),
-		/datum/reagent/drug/aphrodisiac/succubus_milk = list(5, 2 SECONDS),
-		/datum/reagent/drug/aphrodisiac/incubus_draft = list(5, 2 SECONDS),
+		/datum/reagent/drug/aphrodisiac/crocin = list(5, 2 SECONDS, TRUE),
+		/datum/reagent/drug/aphrodisiac/crocin/hexacrocin = list(5, 5 SECONDS, TRUE),
+		/datum/reagent/drug/aphrodisiac/succubus_milk = list(5, 2 SECONDS, TRUE),
+		/datum/reagent/drug/aphrodisiac/incubus_draft = list(5, 2 SECONDS, TRUE),
 )
+	var/static/list/milkable_venoms = generate_milkable_venom_list()
+	var/static/milkable_string = generate_milkable_venom_string()
 
+/proc/generate_milkable_venom_list()
+	RETURN_TYPE(/list)
+
+	var/list/milkable = list()
+
+	for (var/datum/reagent/typepath as anything in /datum/preference/choiced/venomous_bite_venom::venomous_bite_choice_specs)
+		var/list/spec = /datum/preference/choiced/venomous_bite_venom::venomous_bite_choice_specs[typepath]
+
+		if (spec[3])
+			milkable += typepath
+
+	return milkable
+
+/proc/generate_milkable_venom_string()
```

Comment:
this does feel a little sketch, generating it statically instead of dynamically, but realistically this list is never changing